packages/namekit-react/src/components/CurrencySymbol.tsx

```diff
@@ -133,9 +135,9 @@ const getCurrencySymbology = ({
 
 export const CurrencySymbol = ({
   currency,
-  iconSize = CurrencySymbolSize.Small,
-  describeCurrencyInTooltip = true,
-  symbology = CurrencySymbology.Symbol,
+  iconSize = CurrencyIconSize.Small,
+  describeCurrencyInTooltip = false,
+  symbology = CurrencySymbology.TextSymbol,
   ...props
 }: CurrencySymbolProps): JSX.Element => {
   const symbologyToDisplay = (
```

Comment:
This comment is for the `aria-label` a few lines below this. I notice that we incorrectly hardcoded that to always be `Currency.Eth` and not to use the specific `currency` that was provided.